docs/src/main/tut/resources_for_learners.md

```diff
@@ -32,4 +32,7 @@ position: 30
    is in basic Haskell/C++ but the course is designed for programmers of all 
    background to follow. 
 * [Easy and efficient data validation with Cats](https://www.youtube.com/watch?v=P8nGAo3Jp-Q) by [Daniela Sfregola](https://danielasfregola.com/)  this talk introduces Cats, and some of its most interesting components for data validation. In particular, we’ll discuss some options to achieve efficient and expressive data validation. We will also argue that, compared to other options in the language, Cats is particularly suited for the task thanks to its easy-to-use data types and more approachable syntax.
+
+# Social network
+* [@impurepics](https://twitter.com/impurepics) regularly twit informative and kawaii educational pieces about Cats and FP in Scala.  
```

Comment:
Should this be "tweet" instead of "twit"? This is probably me not keeping up with the times.

Reply:
oops, I merged from phone and missed your comment. I'll correct that in another PR.